core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -242,6 +253,17 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
         }
     );
 
+    readonly editorToolbarData$ = this.select(this.editorState$, (editorState) => ({
+        ...editorState,
+        showWorkflowActions:
+            editorState.editorData.mode === EDITOR_MODE.EDIT ||
+            editorState.editorData.mode === EDITOR_MODE.INLINE_EDITING,
+        showInfoDisplay:
+            !editorState.editorData.canEditPage ||
+            (editorState.editorData.mode !== EDITOR_MODE.EDIT &&
+                editorState.editorData.mode !== EDITOR_MODE.INLINE_EDITING)
+    }));
+
```

Comment:
We can also edit variants and we have a mode and a property for it. Check if we need that too.

Reply:
Added the validation!